src/QsCompiler/CompilationManager/EditorSupport/CodeActions.cs

```diff
@@ -38,24 +39,33 @@ private static WorkspaceEdit GetWorkspaceEdit(this FileContentManager file, para
         }
 
         /// <summary>
-        /// Returns all namespaces in which a callable with the name of the symbol at the given position in the given file belongs to.
-        /// Returns an empty collection if any of the arguments is null or if no unqualified symbol exists at that location.
-        /// Returns the name of the identifier as out parameter if an unqualified symbol exists at that location.
+        /// Returns all namespaces in which a callable with the name of the symbol at the given position in the given
+        /// file belongs to.
+        /// 
+        /// Returns an empty collection if any of the arguments is null, if no unqualified symbol exists at that
+        /// location, or if the position is not part of a namespace.
+        /// 
+        /// Returns the name of the identifier as an out parameter if an unqualified symbol exists at that location.
         /// </summary>
         private static IEnumerable<NonNullable<string>> NamespaceSuggestionsForIdAtPosition
             (this FileContentManager file, Position pos, CompilationUnit compilation, out string idName)
         {
             var variables = file?.TryGetQsSymbolInfo(pos, true, out CodeFragment _)?.UsedVariables;
             idName = variables != null && variables.Any() ? variables.Single().Symbol.AsDeclarationName(null) : null;
-            return idName != null && compilation != null
+            var nsName = file.TryGetNamespaceAt(pos);
```

Comment:
This additional query for the namespace is no longer needed (it was needed with private I believe). I'd suggest to remove it again for performance reasons.